apps/aens/test/aens_utils_test.erl

```diff
@@ -13,15 +13,13 @@ to_ascii_test() ->
     ?assertEqual({error, no_registrar},
                  aens_utils:to_ascii(<<"abcdefg">>)),
     ?assertEqual({error, multiple_namespaces},
-                 aens_utils:to_ascii(<<"abcd.efgh.test">>)),
```

Comment:
I guess, <<"bacd.efgh-test">> test should also still fail with this error, but is no longer tested for...
I expected tests to be added, not so much to be removed

Reply:
The diff is a bit confusing here, this test is still present on the branch:

```    
?assertEqual({error, multiple_namespaces},
            aens_utils:to_ascii(<<"abcd.efgh.aet">>)),
```

We can't check registrar of `bacd.efgh-test` thought, for the same reason as mention above.